### PR TITLE
make `mc` relocatable

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

**Comment by mxcl** [comment]
Refs #54

**Comment by mxcl** [comment]
Doesn't actually fix it tho:

```sh
$ tea -X mc                                             
Error opening terminal: xterm-256color.
```

Edit fixed by: https://github.com/teaxyz/pantry.core/commit/ede24811509b562ebbc2de1909adb34495351f09

**Comment by mxcl** [comment]
I don’t understand the failure https://github.com/teaxyz/pantry.extra/actions/runs/3676191307/jobs/6216650678#step:9:94

```
test '"/Users/builder/actions-runner/_work/pantry.extra/pantry.extra/midnight-commander.org/v4.8.28/etc/mc/"' = /Users/builder/actions-runner/_work/pantry.extra/pantry.extra/midnight-commander.org/v4.8.28/etc/mc/
```

If you remove the single quotes the above passes. What I don’t get is *what* is adding the single quotes? @jonchang @jhheider

**Comment by jhheider** [comment]
I think subshell output is implicitly quoted (or treated as a single input argument, possibly). Try `test $(mc --datadir | cut -d' ' -f1) = "{{prefix}}/etc/mc/"`.

**Comment by mxcl** [comment]
Unfortunately made no difference.

**Comment by mxcl** [comment]
bug was due to bug in tea/cli adding extra quotes (lol)

**Comment by mxcl** [comment]
note this is still not fully relocatable, we will need to do similar hacks to those we did for openssl

File: projects/midnight-commander.org/package.yml [hunk 1]
```diff
@@ -54,5 +54,5 @@ build:
 
 # FIXME: hard to test interactive tools
 test:
-  test "$(mc --datadir | cut -d' ' -f1)" = "{{prefix}}/etc/mc/"
-  # ^^ tests relocatability
+  test "$(mc --datadir | cut -wf1)" = "{{prefix}}/etc/mc/"
```
Comment (jhheider) [inline projects/midnight-commander.org/package.yml]: linux cut doesn't have -w

Comment (jhheider) [reply projects/midnight-commander.org/package.yml]: And --datadir can return an array. Hence: `test "$(mc --datadir | cut -d ' ' -f1 | cut -d: -f1)" = "{{prefix}}/etc/mc/"`

**Pull request merged**